### PR TITLE
Attempt to fix System limit for number of file watchers reached

### PR DESCRIPTION
**Pull request by punkle**
The e2e are failing often with the error System limit for number of file watchers reached.

This is an attempt to fix it.

### CODE REVIEW

File: .github/workflows/e2e-tests.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v2
         with:
-          start: yarn dev
+          start: CI=true yarn dev
```
Comment (dtuite) [inline .github/workflows/e2e-tests.yml:43]: Would it be better to set `CI=true` for the whole workflow rather than just this one command?

**Review by dtuite — APPROVED** [review]

File: .github/workflows/e2e-tests.yml (hunk 1 above)
Comment (punkle) [reply .github/workflows/e2e-tests.yml:43]: Maybe yeah. I'm stabbing in the dark. :/

Comment (punkle) [reply .github/workflows/e2e-tests.yml:43]: Its already set. :(

Comment (dtuite) [reply .github/workflows/e2e-tests.yml:43]: So it is.

**Pull request closed without merging**